components/tyk-pump/templates/deployment-pmp.yaml

```diff
@@ -337,6 +356,14 @@ spec:
             value: "4000"
           - name: TYK_PMP_PURGEDELAY
             value: "2"
+        {{- if .Values.pump.service.healthEnabled }}
```

Comment:
You can put both environment variables under single `if` block since conditions are same.